script.js

```diff
@@ -0,0 +1,20 @@
+const hamburger = document.getElementById('handberga-menu');
+const menu = document.getElementById('menu');
+const existBtn = document.getElementById('exit');
+
+function removeClass() {
+  menu.style.display = 'none';
+  existBtn.style.display = 'none';
+  hamburger.style.display = 'block';
+}
+
+function showClass() {
+  hamburger.style.display = 'none';
+  menu.style.display = 'block';
+  existBtn.style.display = 'block';
+}
+
+existBtn.addEventListener('click', removeClass);
+menu.addEventListener('click', showClass);
+hamburger.addEventListener('click', showClass);
+menu.addEventListener('click', removeClass);
```

Comment:
- [ ]  **Kindly note:** Even though this was not requested of you by the previous reviewer, kindly note that for this specific project you were required to update your Linters setup to include eslint for your JavaScript code checks. I strongly recommend that you add the before going forward as it will be required of you in the following project milestones. Kindly update your linters by following the steps provided in [this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

![image](https://user-images.githubusercontent.com/86318284/188618721-9c86eda2-3a07-438c-b72d-6008938a3d03.png)

Reply:
![image](https://user-images.githubusercontent.com/74084998/188648478-baeca24b-dd35-4593-82f7-0b170ae989d7.png)
I already install the linters

Reply:
Hello @anyepeter . The reason why I requested this change is that your linters.yml file was not updated. It does not have the configuration for checking the javascript code, this is Eslint. This said, Have a look at your [Linters.yml](https://github.com/anyepeter/Portfolio-template/blob/Mobile-menu/.github/workflows/linters.yml) file as at the time I requested the change and the correct [Linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file as per the project requirements and I would still suggest you to make the very same changes I requested.